applications/ShapeOptimizationApplication/python_scripts/analyzer_empty.py

```diff
@@ -0,0 +1,21 @@
+# ==============================================================================
+#  KratosShapeOptimizationApplication
+#
+#  License:         BSD License
+#                   license: ShapeOptimizationApplication/license.txt
+#
+#  Main authors:    Baumgaertner Daniel, https://github.com/dbaumgaertner
+#
+# ==============================================================================
+
+# Making KratosMultiphysics backward compatible with python 2.6 and 2.7
+from __future__ import print_function, absolute_import, division
+
+# ==============================================================================
+
+class EmptyAnalyzer( (__import__("analyzer_base")).AnalyzerBaseClass ):

```

Comment:
Interesting idea, i would have solved it with ```None``` and then checking in the main analyzer class, but this is very descriptive and avoids a lot of ```if````s